test/fixtures/wordembeds.json

```diff
@@ -70,31 +68,33 @@
       "children": [
         {
           "type": "text",
-          "value": "And another example with a QR code: "
+          "value": "And another example with a QR code:"
         }
       ]
     },
     {
-      "type": "paragraph",
+      "type": "embed",
```

Comment:
@tripodsan there are a few `paragraph`s that come out as `embed`s in with r@9 – is that intended?

Reply:
don't know :-) what should be the correct behaviour ?

Reply:
so the MD is something like:

```
[![Lars explains the future.](https://hlx.blob.core.windows.net/external/aa59825575016ba5bb9f1ee54be5d7efc23e3032#image.png)](https://www.youtube.com/watch?v=85UKLsvQEIA) 
```

which is correctly detected as `iaEmbed`
maybe this was broken before?

Reply:
I think it was either broken before, now it's good.